src/libs/actions/OnyxUpdates.ts

```diff
@@ -72,8 +72,8 @@ function apply({lastUpdateID, type, request, response, updates}: Merge<OnyxUpdat
 function apply({lastUpdateID, type, request, response, updates}: OnyxUpdatesFromServer): Promise<void | Response> | undefined {
     Log.info(`[OnyxUpdateManager] Applying update type: ${type} with lastUpdateID: ${lastUpdateID}`, false, {command: request?.command});
 
-    if (lastUpdateID && lastUpdateIDAppliedToClient && Number(lastUpdateID) < lastUpdateIDAppliedToClient) {
-        Log.info('[OnyxUpdateManager] Update received was older than current state, returning without applying the updates', false);
+    if (lastUpdateID && lastUpdateIDAppliedToClient && Number(lastUpdateID) <= lastUpdateIDAppliedToClient) {
+        Log.info('[OnyxUpdateManager] Update received was older or the same than current state, returning without applying the updates', false);
```

Comment:
Technically this should be written like this, but NAB since it's a tiny change

```suggestion
        Log.info('[OnyxUpdateManager] Update received was older than or the same as current state, returning without applying the updates', false);
```